### PR TITLE
allow users to disaggregate regional lookups

### PR DESCRIPTION
**Pull request by Broham**

### CODE REVIEW

**Comment by arnoldbm** [comment]
+1 looks good

**Comment by jgoley** [comment]
I made a few styling updates, disabled the _disaggregate_ button if no meta nodes are selected and some refactoring or code-style updates here and there.

**Comment by jgoley** [comment]
Oh wait...just realized there's now a visual bug with button padding. Fixing...

**Comment by jgoley** [comment]
@Broham This should be good to go after you look over my changes. Specifically 980c08a

File: app/packages/grits-net-meteor/client/templates/grits_dataTable.jade [hunk 1]
```diff
@@ -24,25 +27,25 @@ template(name='gritsDataTable')
...
+                  td.dtHidden {{getNodeCountry this.origin}}
+                  td          {{this.destination._id}}
+                  td.dtHidden {{getNodeName this.destination}}
+                  td.dtHidden {{getNodeCity this.destination}}
+                  td.dtHidden {{getNodeState this.destination}}
+                  td.dtHidden {{getNodeCountry this.destination}}
+                  td          {{this.occurrences}}
+                  td          {{this.throughput}}
+                    .circle(style='background-color: {{getPathThroughputColor this}}')
+              else
```
Comment (Broham) [inline app/packages/grits-net-meteor/client/templates/grits_dataTable.jade:45]: This is cool - I didn't realize that you could use an `else` to handle empty collections

**Comment by Broham** [comment]
👍  - looks good.  Going to merge unless @nathanathan wants to take a look.

**Pull request merged**